dev-common-postgres-image/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM postgres:13-buster
 
-ENV POSTGRES_USER postgres
+ENV POSTGRES_USER appuser
```

Comment:
Should this user be `postgres`? It'll be doing create schemas which this user would be able to do